client/utils/reduxFormUtils.js

```diff
@@ -54,50 +54,6 @@ export function validateSettings(formProps) {
   return errors;
 }
 
-export function validateLogin(formProps) {
-  const errors = {};
-  if (!formProps.email) {
-    errors.email = i18n.t('ReduxFormUtils.errorEmptyEmail');
-  }
-  if (!formProps.password) {
-    errors.password = i18n.t('ReduxFormUtils.errorEmptyPassword');
-  }
-  return errors;
-}
-
-function validatePasswords(formProps, errors) {
-  if (!formProps.password) {
-    errors.password = i18n.t('ReduxFormUtils.errorEmptyPassword');
-  }
-  if (formProps.password && formProps.password.length < 6) {
-    errors.password = i18n.t('ReduxFormUtils.errorShortPassword');
-  }
```

Comment:
You can see how the deleted code has specific translated messages for various failures.  We need to integrate those into the Zod code.  Not sure if this is the right syntax but it would be something along the lines of `.min(6, i18n.t('ReduxFormUtils.errorShortPassword'))`.